sdk/appplatform/Azure.ResourceManager.AppPlatform/src/Generated/Models/ResourceSku.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.AppPlatform.Models
+{
+    /// <summary> Describes an available Azure Spring Cloud SKU. </summary>
+    public partial class ResourceSku
```

Comment:
`ResourceSku` seems to be too general and might cause confusion with the sku in Resources RP.

Reply:
Rename to `AvailableAppPlatformSku`